app/resources/file_sets/file_sets_controller.rb

```diff
@@ -8,6 +8,7 @@ class FileSetsController < ApplicationController
     metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister),
     storage_adapter: Valkyrie::StorageAdapter.find(:disk)
   )
+  before_action :parent_resource, only: [:destroy]
 
   def derivatives
     file_set = find_resource(params[:id])
```

Comment:
there's one more here! I think you can just delete this line.

Reply:
Thanks!

Reply:
@hackmastera removed!